From grant.likely@secretlab.ca  Mon Mar 15 15:22:35 2010
From: Grant Likely <grant.likely@secretlab.ca>
Date: Thu, 11 Mar 2010 15:06:54 -0700
Subject: usb/gadget: fix compile error on r8a66597-udc.c
To: gregkh@suse.de, linux-usb@vger.kernel.org, linux-kernel@vger.kernel.org
Message-ID: <20100311220631.19163.98586.stgit@angua>


C file uses IS_ERR and PTR_ERR, but doesn't include <linux/err.h>

Signed-off-by: Grant Likely <grant.likely@secretlab.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/r8a66597-udc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/r8a66597-udc.c
+++ b/drivers/usb/gadget/r8a66597-udc.c
@@ -23,6 +23,7 @@
 #include <linux/module.h>
 #include <linux/interrupt.h>
 #include <linux/delay.h>
+#include <linux/err.h>
 #include <linux/io.h>
 #include <linux/platform_device.h>
 #include <linux/clk.h>
